### DISCUSSION TITLE
gpclient makes 2 successful SAML requests then logs out

### DISCUSSION THREAD

**Discussion by gitBrikski** [yuezk/GlobalProtect-openconnect#325] [General]
Sorry, saw a similar discussion before but now can't find it.
gpclient 2.0.0 (2024-02-05) makes two successful SAML authorization requests, but then exits with logout.   Trial version of GUI connects successfully.  This is on Fedora 39, transcript of the command window messages attached.  Nice tool overall, hope you can fix this bug easily.
[gpclientFail_02-19-23.log](https://github.com/yuezk/GlobalProtect-openconnect/files/14337101/gpclientFail_02-19-23.log)

**Comment by yuezk** [comment]
@gitBrikski You should run the command with `sudo`.
